### ISSUE TITLE
One per language fields

### ISSUE THREAD

**Issue by woodkri** [cassproject/cass-editor#495]
Don't allow multiple entries with the same language in these fields. Show alert and revert language to something else.

Related: Determine what to default language to for these fields.

**Comment by woodkri** [comment]
Thinking about this further, I think it makes more sense to highlight the incorrect fields for the user to change rather than change them to something else automatically.

At the moment there's just one default language for the whole framework and that's what is displayed even for these fields that are limited to one entry per language. It would make more sense to me not to put a default language for duplicate entries in those fields since a good option couldn't be easily determined. The way it is currently may be the best option though if we want something to show up in that field no matter what.

**Comment by eddavies** [comment]
Working 👍. Just have two minor usability concerns:
The prompt/error message has to be closed twice instead of once. 
Should we also have the field highlight in orange like we do with errors such as a field requiring a uri (we don't have to, just opening up the suggestion)?

**Comment by stuartasutton** [comment]
Can we verify here which fields this work considers 1 language per field?

**Comment by woodkri** [comment]
@stuartasutton This is based on the CASS-ASN mapping spreadsheet. Any fields that are designated one per language there will no longer allow you to save two things with the same language (two different framework names with "en-US", for example).

**Comment by stuartasutton** [comment]
Thanks. Just checking...

**Comment by woodkri** [comment]
@eddavies You shouldn't have to close the alert twice, I'm wondering if that happened to due a double click. I disabled the double click for the save button, let me know if the alert is still popping twice in case something else is causing that!

I also added instruction text to the tooltips for the fields that are one per language. The invalid fields should be highlighted red after you close the alert; is that not happening on your end?
